### DISCUSSION TITLE
Why does not deleting the /var/lib/rook folder cause RBD permission denied error in rook ?

### DISCUSSION THREAD

**Discussion by CrossainQi** [rook/rook#12432] [Q&A]
I delete my old ceph cluster without deleting /var/lib/rook directory and I faced "RBD permission denied" error in operator's log during deploying a new cluster. I found the content of /var/lib/rook/mon-a/data/keyring is different from which in rook-ceph-mons-keyring. I know the steps to solve this problem but I'm curious about who uses the file /var/lib/rook/mon-a/data/keyring or at what step the checksum is done？

**Comment by travisn** [comment]
After you re-created the cluster, the mons should have failed before they ever formed quorum, which usually leads to the `ceph` commands failing and timing out in the operator. The specific error for "rbd permissions denied" sounds different than I would expect, but in any case the operator would be blocked from proceeding until the mons can properly start and form quorum. Thus, the [cleanup guide](https://rook.io/docs/rook/latest/Getting-Started/ceph-teardown/#delete-the-data-on-hosts) is critical when reinstalling.
